app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Api::V1::ReviewsController < ApiController 
+  def create
+    review = Review.new(review_params)
+    review[:movie_id] = params["movie_id"]
```

Comment:
Ruby and ActiveRecord like when we pass whole objects where possible - here with the movie_id, we should also make sure that it's a movie record in the database
```ruby
review.movie = Movie.find(params[:movie_id])
```